src/message/message-parser.js

```diff
@@ -10,16 +11,7 @@ const C = require('../constants/constants')
  *
  * @returns {Object} actions
 */
-const actions = (function getActions () {
-  const result = {}
-  let key
-
-  for (key in C.ACTIONS) {
-    result[C.ACTIONS[key]] = key
-  }
-
-  return result
-}())
+const actions = new Set(Object.keys(C.ACTIONS).map(key => C.ACTIONS[key]))
```

Comment:
Yeah figured that might the case. Just had this discussion with the core team. A JS perf would be brilliant!